{"name": "test_mul[get_datas_mul3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <pratice_demo.pratice_calculator_advanced.test_calc_new.TestCalculator object at 0x0000024314FBA0D0>\nget_calc = <pratice_demo.pratice_calculator_advanced.demo.calc.Calculator object at 0x0000024314E8D550>, get_datas_mul = [-1, -999999999.1, 999999999.1]\n\n    @allure.story(\"计算器的乘法测试\")\n    @pytest.mark.run(order=3)\n    def test_mul(self, get_calc, get_datas_mul):\n        if get_datas_mul[0] > 999999999 or get_datas_mul[1] > 999999999:\n            print(\"超出最大数值999999999，不允许输出\")\n            assert False\n        # 假设最小能输入-999999999\n        elif get_datas_mul[0] < -999999999 or get_datas_mul[1] < -999999999:\n            print(\"超出最小数值-999999999，不允许输出\")\n>           assert False\nE           assert False\n\ntest_calc_new.py:174: AssertionError"}, "attachments": [{"name": "stdout", "source": "dc8b5bd0-f964-496d-8e0a-3407cf7725c1-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_datas_mul", "value": "[-1, -999999999.1, 999999999.1]"}], "start": 1612345709716, "stop": 1612345709716, "uuid": "468dc9a8-3460-4a37-88da-e07986a34b45", "historyId": "6d3fb21b3dcfa1bf8a94869dee189c5e", "testCaseId": "bf91843f7e3da214fab11bfe24640dc5", "fullName": "test_calc_new.TestCalculator#test_mul", "labels": [{"name": "story", "value": "计算器的乘法测试"}, {"name": "feature", "value": "测试计算器"}, {"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "Naraku15"}, {"name": "thread", "value": "10164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_new"}]}